### ISSUE TITLE
Docker Maven JAVA_HOME 

### ISSUE THREAD

**Issue by vaughnmb** [carlossg/docker-maven#282]
Using the latest version of the maven Docker image, I now get this error:  The JAVA_HOME environment variable is not defined correctly.

I never encountered this issue with previous versions of the Maven Docker image.  Is this by design and something I need to handle on my end?

**Comment by gmolkvk** [comment]
May be related to https://github.com/carlossg/docker-maven/issues/281 ?

**Comment by carlossg** [comment]
what image tag, what command,... are you trying to use

**Comment by carlossg** [comment]
>  what image tag, what command,... are you trying to use

**Comment by vaughnmb** [comment]
I am trying to use the maven:latest.   The command was "mvn clean package"

**Comment by carlossg** [comment]
without more details I cannot help.

This works in this repo

```
docker run -it --rm -v "$(pwd)"/tests:/usr/src/mymaven -w /usr/src/mymaven maven:latest mvn clean package
```

**Comment by mpaktheone** [comment]
what details do you need?
I have Dockerfile with content:
FROM maven:latest as build
WORKDIR /app
COPY mvnw .
COPY .mvn .mvn
COPY pom.xml .
RUN mvn build clean
....

and when im do:
docker build --rm --build-arg VERSION=test --tag test:test .

i get this error:
Step 6/57 : RUN mvn build clean
 ---> Running in 5587a9575359
The JAVA_HOME environment variable is not defined correctly,
this environment variable is needed to run this program.

if i change Dockerfile to
FROM maven:3.8.4 as build
WORKDIR /app
COPY mvnw .
COPY .mvn .mvn
COPY pom.xml .
RUN mvn build clean
....

all build successful
Problem start at may 30, because at may 27 everything was good.

**Comment by carlossg** [comment]
what's the output of 

```
docker run --rm -it maven:latest bash -c 'env | grep JAVA_HOME; java -version; mvn -version'
docker pull maven:latest
docker run --rm -it maven:latest bash -c 'env | grep JAVA_HOME; java -version; mvn -version'
```

**Comment by carlossg** [comment]
yeah, so it is the same as https://github.com/carlossg/docker-maven/issues/281

**Comment by mpaktheone** [comment]
> yeah, so it is the same as #281

it will be fixed?

**Comment by carlossg** [comment]
you have the answer in https://github.com/adoptium/containers/issues/215#issuecomment-1142046045
TL;DR upgrade Docker

**Comment by mpaktheone** [comment]
> you have the answer in [adoptium/containers#215 (comment)](https://github.com/adoptium/containers/issues/215#issuecomment-1142046045) TL;DR upgrade Docker

thx, it work again!
